AGENTS.md

```
@@ -395,13 +396,13 @@ src/contexts/<context>/modules/<feature>/
 ```
 
 3. Wire the module:
-   - Register Mongoose schema in module imports.
-   - Register controller.
-   - Register use cases and repository provider with injection token.
+  - Register Mongoose schema in module imports.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)